drivers/sensor/bq274xx/bq274xx.c

```diff
@@ -0,0 +1,603 @@
+/*
+ * Copyright (c) 2020 Linumiz
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <drivers/i2c.h>
+#include <init.h>
+#include <drivers/sensor.h>
+#include <sys/__assert.h>
+#include <string.h>
+#include <sys/byteorder.h>
+
+#include "bq274xx.h"
+
+#if defined(DT_INST_0_TI_BQ274XX_DESIGN_VOLTAGE)
+#define BQ274XX_DESIGN_VOLTAGE		DT_INST_0_TI_BQ274XX_DESIGN_VOLTAGE
+#else
+#define BQ274XX_DESIGN_VOLTAGE		3700
```

Comment:
You can set a `default` in the DT binding instead. See https://docs.zephyrproject.org/latest/guides/dts/bindings.html#bindings-file-syntax

Reply:
Adapter, thanks for pointing it.